fix(lsm): Fixes for TigerBeetle LSM updates

koekiebox opened the pull request:
### Updates include:
* Initial changes for 'lsm' fixes.
* Updated error codes for create account and transfer.

koekiebox committed:
Changes include:
* Initial changes for 'lsm' fixes.
* Updated error codes for create account and transfer.

jorangreef commented on src/index.ts:
```diff
@@ -48,6 +48,7 @@ export enum CreateAccountError {
   reserved_field,
 
   id_must_not_be_zero,
+  id_must_not_be_max_int,
```
```suggestion
  id_must_not_be_int_max,
```

We should change this upstream as well—would you please do that too @koekiebox ?

koekiebox replied:
@jorangreef  Done. https://github.com/coilhq/tigerbeetle/pull/131

jorangreef reviewed (commented):
Thanks @koekiebox !

koekiebox committed:
Changes include:
* Initial changes for 'lsm' fixes.
* Updated error codes for create account and transfer.

koekiebox committed:
Changes include:
* Updated command for tigerbeetle benchmark.sh

kprotty commented on src/node.zig:
```diff
@@ -138,26 +139,19 @@ const Context = struct {
-        );
-        errdefer context.message_bus.deinit();
-
+        var message_pool = try MessagePool.init(allocator, .client);
```
The client holds an address to the MessagePool, but the message pool is invalidated once this function returns.
Try storing the MessagePool inside the Context, making sure to also deinit() it later.

sentientwaffle committed:
node.zig: Update for MessagePool refactor

`MessagePool` is owned by the `Context`, so it must not be allocated on the stack.

sentientwaffle reviewed (changes requested):
Sorry, approved the wrong PR!

koekiebox committed:
Changes include:
* README.md - Update for latest commands.

koekiebox committed:
Changes include:
* Bump submodule file.

koekiebox committed:
Changes include:
* README.md - Updated commands.

koekiebox committed:
Changes include:
* README.md - Updated commands.

koekiebox committed:
Changes include:
* README.md - Updated commands.

sentientwaffle reviewed (approved):
LGTM.
